.github/workflows/industrial_ci_action.yml

```diff
@@ -0,0 +1,105 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: CI
```

Comment:
please rename all these files to have the `.yaml` file extension instead of `.yml`.  Also make this file just `ci.yaml` and make sure you update the readme to point to the new filenames.